chaos_genius/controllers/digest_controller.py

```diff
@@ -59,6 +59,8 @@ def preprocess_triggered_alert(
     if not isinstance(alert_conf.alert_channel_conf, dict):
         triggered_alert.alert_channel_conf = None
     else:
```

Comment:
we can add a  TODO marker to find this for future refactor , what do u think?

Reply:
i have added one here - https://github.com/chaos-genius/chaos_genius/blob/d0d2b7f6ed1cbe6862397486c8c742a9aa023553/chaos_genius/alerts/anomaly_alerts.py#L208